nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -229,60 +229,61 @@ def _get_subgraph_data_for_node(self, node: NNCFNode, nncf_graph: NNCFGraph) ->
         statistic_nodes = []
         subgraph_input_ids, subgraph_output_ids = [], []
 
-        def fill_subgraph_output_ids(edge: NNCFGraphEdge):
-            node = edge.to_node
-            if node in visited_nodes:
-                return
-            visited_nodes.append(node)
-
-            input_id = (node.node_name, edge.input_port_id)
-            output_id = (edge.from_node.node_name, edge.output_port_id)
-
-            # If we found a node with bias, we have to collect it as a statistic node,
-            # and its input for _collected_stat_inputs_map,
-            # which will be used during the collection of statistics for the next node.
-            if self._is_node_correctable(node, nncf_graph):
-                subgraph_output_ids.append(output_id)
-                self._collected_stat_inputs_map[input_id] = output_id
-
-                statistic_nodes.append(edge.from_node)
-                return
-
-            for output_edge in nncf_graph.get_output_edges(node):
-                fill_subgraph_output_ids(output_edge)
-
-        def fill_subgraph_input_ids(edge: NNCFGraphEdge):
-            node = edge.from_node
-            input_id = (edge.to_node.node_name, edge.input_port_id)
-
-            # Since we need to find the inputs for the subgraph,
-            # we can take only those layers for which we have already collected statistics.
-            if input_id in self._collected_stat_inputs_map:
-                activation_id = self._collected_stat_inputs_map[input_id]
-                if activation_id not in subgraph_output_ids:
-                    subgraph_input_ids.append(input_id)
-                    return
+        def fill_subgraph_output_ids(main_node: NNCFNode):
```

Comment:
Could you add small descriptions of these functions?

Reply:
These methods aren't new. There are comments that exists for `_get_subgraph_data_for_node` at all.

Reply:
It will make the understanding of these functions easier. Also, it will help to understand whether these two functions can be simplified and merged into one.

Reply:
Done.